test-app/entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+poetry install
+poetry run pytest
```

Comment:
is this used?

Reply:
no, I was gonna have Dockerfile copy that in and run it for the tests. and then I forgot about it 😅